web/packages/plugins/shared/common/src/components/Pickers/DestinationPicker.tsx

```diff
@@ -1,128 +1,71 @@
-      onChange(nodePath)
-    } else {
-      onChange(dataSource + '/' + nodeId)
-    }
   }
 
   return (
     <>
-      {/*<b>{title}</b>*/}
-      {/*<div>*/}
-      {/*  <input*/}
-      {/*    style={{ width: '100%' }}*/}
-      {/*    type="string"*/}
-      {/*    value={destination}*/}
-      {/*    readOnly={true}*/}
-      {/*    onClick={() => setShowModal(true)}*/}
-      {/*  />*/}
-      {/*  <Modal*/}
-      {/*    toggle={() => setShowModal(!showModal)}*/}
-      {/*    open={showModal}*/}
-      {/*    title={'Select a folder as destination'}*/}
-      {/*  >*/}
-      {/*    <Tree*/}
-      {/*      state={index.models.tree.models.tree}*/}
-      {/*      operations={index.models.tree.operations}*/}
-      {/*    >*/}
-      {/*      {(renderProps: TreeNodeRenderProps) => {*/}
-      {/*        const { actions, nodeData } = renderProps*/}
-      {/*        const type = nodeData.meta.type*/}
-
-      {/*        return (*/}
-      {/*          <div style={{ display: 'flex' }}>*/}
-      {/*            {renderProps.iconGroup(() => {*/}
-      {/*              handleOpenOrExpand(renderProps)*/}
-      {/*            })}*/}
-      {/*            <div onClick={() => handleOpenOrExpand(renderProps)}>*/}
-      {/*              {nodeData.title}*/}
-      {/*              {nodeData.isLoading && (*/}
-      {/*                <small style={{ paddingLeft: '15px' }}>Loading...</small>*/}
-      {/*              )}*/}
-      {/*            </div>*/}
-      {/*            {type === blueprintFilter && (*/}
-      {/*              <div*/}
-      {/*                style={{*/}
-      {/*                  display: 'flex',*/}
-      {/*                  width: '100%',*/}
-      {/*                  flexFlow: 'row-reverse',*/}
-      {/*                }}*/}
-      {/*              >*/}
-      {/*                <SelectDestinationButton*/}
-      {/*                  onClick={() => {*/}
-      {/*                    onSelect(*/}
-      {/*                      nodeData.nodeId,*/}
-      {/*                      `${renderProps.path}/${nodeData.title}`*/}
-      {/*                    )*/}
-      {/*                  }}*/}
-      {/*                >*/}
-      {/*                  Select*/}
-      {/*                </SelectDestinationButton>*/}
-      {/*              </div>*/}
-      {/*            )}*/}
-      {/*          </div>*/}
-      {/*        )*/}
-      {/*      }}*/}
-      {/*    </Tree>*/}
-      {/*  </Modal>*/}
-      {/*</div>*/}
+      <div style={{ width: '100%' }}>
+        <Input
+          style={{ width: '280px', margin: '0 8px', cursor: 'pointer' }}
+          type="string"
+          value={formData}
+          onClick={() => setShowModal(true)}
+        />
+        <Modal
+          toggle={() => setShowModal(!showModal)}
+          open={showModal}
+          title={'Select a folder as destination'}
+        >
+          <TreeView
+            onSelect={() => {}}
+            // @ts-ignore
```

Comment:
would be nice to remove this ts-ignore if possible

Reply:
done